app-tracking-protection/vpn-impl/src/main/res/layout/activity_report_breakage_category_single_choice.xml

```diff
@@ -41,16 +41,14 @@
 
             <com.google.android.material.textfield.TextInputLayout
                 android:id="@+id/textInputLayout"
-                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
```

Comment:
good catch! this doesn't work as expected because it's not using the correct theming. Will be fixed soon.